index.html

```diff
@@ -170,6 +171,19 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2 id="risks">Risk areas</h2>
+
+<p>There are many potential risk areas for a proposal. Some examples follow, but the committee may specify additional or modified risk areas, or none, as it sees fit. Risk areas should be identified as part of entrance into stage 2, so that champions can be sure to address these as efficiently as possible.</p>
```

Comment:
The language of this paragraph seems to be slightly different from what I remember we agreed upon at the last meeting in the area of responsibility.
At the meeting, the language we used is that the *champion* is responsible for identifying the risk areas and presenting them to the committee which may indicate additional risk areas to be included.

The wording of this paragraph feels a bit like its the committee that identifies the risk areas so that the champion can address them.

I'd also like to suggest to add the risk area verification to the reviewers tasks.

Reply:
Ah, I borrowed the phrasing from "The committee may elide the process based on the scope of a change under consideration as it sees fit." below.

Since this PR makes "identifying risk areas" a stage 2 requirement, it's automatically on the champion to do so - and the committee has to accept them.

We'll discuss this next week and narrow down the consensus :-)